cabal-install/src/Distribution/Client/ProjectPlanning.hs

```diff
@@ -3236,7 +3233,7 @@ pruneInstallPlanPass1
 pruneInstallPlanPass1 pkgs
   -- if there are repl targets, we need to do a bit more work
   -- See Note [Pruning for Multi Repl]
```

Comment:
Is this note still relevant?

Reply:
The last part is out of date, I updated it.